src/prompts/index.tsx

```diff
@@ -126,22 +110,10 @@ export class PromptManager {
         this.hideAndSuppressPrompts(prompt);
 
         if (this.subscriptionState === 'subscribed') {
-            await this.handleUserChannelSelection(channelSelections);
-
             this.ui?.showToast(prompt.labels?.thanksForSubscribing!);
         }
     };
 
-    private onPostActionConfirm = async (
-        prompt: PushPromptConfig,
-        channelSelections?: ChannelListItem[]
-    ) => {
-        await this.handleUserChannelSelection(channelSelections);
-
-        this.setState('ready');
```

Comment:
Is this state transition back to ready now missing?

Do we still need to keep the 'hideAndSuppressPrompts' below as well?

Reply:
(perhaps not, this is about the 'post action', which if I read correctly, we have removed the concept/handling of 'post actions'?)

Reply:
Yes, following the code the currentPostAction is always null so it never renders the ChannelsDialog which called the onPostActionConfirm. 

Perhaps it was half deleted in [this PR](https://github.com/Kumulos/KumulosSdkWeb/pull/57/files)